hack/build-charts.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+
+TMP_PLANTUML_DIR=$(mktemp -d)
+wget -O "${TMP_PLANTUML_DIR}/plantuml.jar" http://sourceforge.net/projects/plantuml/files/plantuml.jar/download
+java -jar "${TMP_PLANTUML_DIR}/plantuml.jar" -checkmetadata -tsvg pkg/controller/hyperconverged/hyperconverged_controller.go
```

Comment:
We can also evaluate a dockerized version: https://hub.docker.com/r/plantuml/plantuml-server
but maybe this is still faster

Reply:
Their docker is an HTTP server, not a command line. It's much simpler, safer and faster to do that like that.